Mirror/Runtime/Messages.cs

```diff
@@ -150,18 +150,21 @@ class CommandMessage : MessageBase
     {
         public uint netId;
         public int cmdHash;
+        public byte componentIndex;
```

Comment:
what about byte vs. int here?
weaver will still use 1 byte if the int is <241. and would avoid the byte cast below. and situations where people have >256 components would be less weird. I don't know why anyone would need that many, but I some people in ummorpg add dozens of attributes and energy types etc., who knows.
up to you really. if you think byte is better then I am fine with it

Reply:
in other words: if we use int then we will never have to worry about this no matter what. might be worth it, might be not